crates/tests/src/clique.rs

```diff
@@ -0,0 +1,374 @@
+use enr::k256::ecdsa::SigningKey;
+use ethers_core::{
+    types::{Address, Block, Bytes, U64},
+    utils::{ChainConfig, CliqueConfig, Genesis, GenesisAccount, Geth},
+};
+use ethers_signers::{LocalWallet, Signer};
+use reth_cli_utils::chainspec::{ChainSpecification, Genesis as RethGenesis};
+use reth_consensus::Config as ConsensusConfig;
+use reth_eth_wire::{EthVersion, Status};
+use reth_net_test_utils::unused_port;
+use reth_primitives::{
+    proofs::genesis_state_root, Chain, ForkHash, ForkId, Header, H160, INITIAL_BASE_FEE,
+};
+use std::collections::HashMap;
+
+/// Converts an ethers [`Genesis`] into a reth
+/// [`ChainSpecification`](reth_cli_utils::chainspec::ChainSpecification).
```

Comment:
This should probably be a From implementation and the ChainSpecification should be part of the reth crate I mentioned